### DISCUSSION TITLE
請問v5版的配置文件， PriorityDomain還支持geosite域名集嗎？

### DISCUSSION THREAD

**Discussion by WordsWorthLess** [v2fly/v2ray-core#1653] [Q&A]
看配置文檔介紹
https://www.v2fly.org/v5/config/dns.html#prioritydomainobject
PriorityDomain貌似只有4種匹配方式
完整域名匹配，關鍵字匹配，(子)域名匹配，正則表達式匹配，
而且示例配置也沒有提及geosite域名集的相關配置（geosite.dat路徑，域名集填寫方法等）
是網站的説明漏了提及還是v2ray v5不再支持geosite合集了?如果是後者的話就太令人費解了
而且我覺得原來V4的DNS 服務器列表配置方式就很清爽直觀，在V4的基礎上添加inverseMatch就很好了，V5搞得太繁瑣太臃腫了。

**Comment by zjufrankzhang** [comment]
同样有这个困惑，当时刚上线以为是说明没写完。但这都上线一年了，依然没有这个的支持，导致只能继续用v4的config。
预定义域名列表能够防止敏感域名解析泄露，同时也能提升国内域名的解析速度与准确度。不知道为什么这个功能就默默的被砍掉了

**Reply by maike2019** [reply]
我也一样 啊，我还在用V4配置。你看我的配置对吗？https://github.com/v2fly/v2ray-core/discussions/2571#discussion-5329773

**Comment by raphaelsoul** [comment]
应该是支持的 核心实现代码并没有太大的差异 只是文档还没写 因为v4版本的配置载入的代码实现太混乱了
